flaskr/__init__.py

```diff
@@ -1,6 +1,8 @@
 import os
 
 from flask import Flask
+from movies import bp
+from db import init_app
```

Comment:
won't this cause name conflicts in the future if you add more blueprints

Reply:
That's correct and was an oversight on my part

Reply:
That's correct and was an oversight on my part